python/dask_cudf/dask_cudf/tests/test_core.py

```diff
@@ -12,10 +12,10 @@
 from dask.dataframe.core import make_meta, meta_nonempty
 from dask.utils import M
 
-import cudf
-
 import dask_cudf as dgd
 
+import cudf
+
```

Comment:
This change feels weird, I would expect `dask_cudf` import to be at the bottom in `dask_cudf` code based on `isort`.

Reply:
Yup, fixed this.